libs/shared/assets/styles/theme.css

```diff
@@ -1,5 +1,5 @@
 :root {
-  --brand-01: #00327f;
+  --brand-01: 0, 50, 127;
```

Comment:
using the separated rgb values is much easier to use then the hex value:

in CSS you can use `color: rgb(var(--brand-01))`, yielding the same result as `color: var(--brand-01)` as it was a hex value. But the latter you can't use to apply to opacity, while the first allows you to define `color: rgb(var(--brand-01), 0.25)`, that is a 25% opacity.

Reply:
Moved to a new issues